src/object/srv_obj.c

```diff
@@ -2596,8 +2622,6 @@ ds_obj_tgt_update_handler(crt_rpc_t *rpc)
 
 		if (rc < 0 && rc != -DER_NONEXIST)
 			D_GOTO(out, rc);
-
-		dtx_flags |= DTX_RESEND;
```

Comment:
just curious why don't need it now?

Reply:
DTX_RESEND flag is used to set dth->dth_resent that is useless for a long time. So cleanup them in the patch.